development/site-components/sm-buttons.pug

```diff
@@ -0,0 +1,15 @@
+html
```

Comment:
Please use spaces not tabs, indentation of 2 for this file. Thanks.
 Add a line note